python/paddle/fluid/core.py

```diff
@@ -332,10 +328,6 @@ def to_list(s):
         from .core_noavx import _load_static_dict
         from .core_noavx import _save_dygraph_dict
```

Comment:
remove useless save load function in next PR

Reply:
Done, thx.